doc/dev/index.rst

```diff
@@ -542,6 +542,33 @@ is deemed to be an integration test. Integration tests are organized into
 "suites", which are defined in the `ceph-qa-suite repository`_ and run with
 the `teuthology-suite`_ command.
 
+.. _`teuthology-suite`: http://url/of/teuthology-suite/documentation
+
+The `teuthology-suite`_ command is part of the `teuthology framework`_.
+In the sections that follow we attempt to provide a detailed introduction
+to that framework from the perspective of a beginning Ceph developer.
+
+The most important thing to understand about teuthology
```

Comment:
I would leave this out. Having the **Teuthology consumes packages** first is probably a good indication that it is something important.